Cargo.toml

```diff
@@ -6,39 +6,44 @@ members = ["crates/*"]
 anyhow = "1"
 async-trait = "0.1"
 bigdecimal = "0.3"
-cached = { version = "0.39", default-features = false }
+cached = { version = "0.44", default-features = false }
 chrono = { version = "0.4", default-features = false }
 clap = { version = "4", features = ["derive", "env"] }
 derivative = "2"
 ethcontract = { version = "0.24.0", default-features = false, features = ["aws-kms"] }
 ethcontract-generate = { version = "0.24", default-features = false }
 ethcontract-mock = { version = "0.24", default-features = false }
 futures = "0.3"
-gas-estimation = { git = "https://github.com/cowprotocol/gas-estimation", tag = "v0.7.1", features = ["web3_", "tokio_"] }
+gas-estimation = { git = "https://github.com/cowprotocol/gas-estimation", branch = "update-deps", features = ["web3_", "tokio_"] }
 hex = { version = "0.4", default-features = false }
-hex-literal = "0.3"
-itertools = "0.10"
+hex-literal = "0.4"
```

Comment:
Fun tid-bit, but `hex-literal` is not implemented as a const-fn instead of a proc-macro! Its very cool!